docs/docsite/rst/dev_guide/developing_modules_general_windows.rst

```diff
@@ -190,6 +190,62 @@ A very basic powershell module `win_environment <https://github.com/ansible/ansi
 
 A slightly more advanced module is `win_uri <https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/windows/win_uri.ps1>`_ which additionally shows how to use different parameter types (bool, str, int, list, dict, path) and a selection of choices for parameters, how to fail a module and how to handle exceptions.
 
+As part of the new ``AnsibleModule`` wrapper, the input parameters are defined and validated based on an argument
```

Comment:
Are the parameters defined below specific to Windows modules?

Reply:
Not 100%, they are shared with basic.py but the structure is slightly different.